src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/memberdays/MemberDaysReport.java

```diff
@@ -126,9 +128,21 @@ private String getDatePart(String dateTimeValue) {
                 .filter(x -> x.getValue().getListedDate().split("T")[0]
                     .equals(listingData.getListingDate()))
                 .collect(Collectors.toList());
+        } else {
+            return dateListedTypeItems.stream()
+                .filter(x -> isHearingDateInRange(x.getValue().getListedDate(),
+                        listingData.getListingDateFrom(), listingData.getListingDateTo()))
+                .collect(Collectors.toList());
         }
+    }
+
+    private boolean isHearingDateInRange(String dateListed, String dateFrom, String dateTo) {
+        var hearingListedDate = LocalDate.parse(dateListed.split("T")[0]);
```

Comment:
I'm not sure why you've made the choice here to parse the date by splitting the string rather than using the OLD_DATE_TIME_PATTERN formatter constant?

Reply:
You are right It is one way of doing it. I tested it and your approach works too.